baseframe/utils.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import six
+from six.moves.urllib.parse import quote as urlquote
+
+from mxsniff import mxsniff, MXLookupException
+
+from . import asset_cache
+
+__all__ = ['is_public_email_domain']
+
+
+def is_public_email_domain(email_or_domain, default=None, timeout=30):
+    """
+    Checks if the given domain (or domain of given email) is known to offer public email accounts
+
+    :param email_or_domain: email address or domain name to check
+    :param default: default value to return in case domain lookup fails
```

Comment:
These two descriptions begin with a lowercase letter, while the next one begins with uppercase. Consistency?